dotnet/xml/Microsoft.Identity.Abstractions/ManagedIdentityOptions.xml

```diff
@@ -0,0 +1,96 @@
+<Type Name="ManagedIdentityOptions" FullName="Microsoft.Identity.Abstractions.ManagedIdentityOptions">
+  <TypeSignature Language="C#" Value="public class ManagedIdentityOptions" />
+  <TypeSignature Language="ILAsm" Value=".class public auto ansi beforefieldinit ManagedIdentityOptions extends System.Object" />
+  <TypeSignature Language="DocId" Value="T:Microsoft.Identity.Abstractions.ManagedIdentityOptions" />
+  <TypeSignature Language="VB.NET" Value="Public Class ManagedIdentityOptions" />
+  <TypeSignature Language="F#" Value="type ManagedIdentityOptions = class" />
+  <AssemblyInfo>
+    <AssemblyName>Microsoft.Identity.Abstractions</AssemblyName>
+    <AssemblyVersion>5.1.0.0</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces />
+  <Docs>
+    <summary>
+            Data object to hold the definition of a managed identity for an application to use for authentication. If 
+            <see cref="P:Microsoft.Identity.Abstractions.ManagedIdentityOptions.UserAssignedClientId" /> is null, the application will use the system-assigned managed identity. If
+            <see cref="P:Microsoft.Identity.Abstractions.ManagedIdentityOptions.UserAssignedClientId" /> is set, the application will try to use the user-assigned managed identity associated
+            with the provided ClientID. See <see href="https://aka.ms/Entra/ManagedIdentityOverview" /> for more details.
+            </summary>
+    <remarks>To be added.</remarks>
+  </Docs>
+  <Members>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public ManagedIdentityOptions ();" />
+      <MemberSignature Language="ILAsm" Value=".method public hidebysig specialname rtspecialname instance void .ctor() cil managed" />
+      <MemberSignature Language="DocId" Value="M:Microsoft.Identity.Abstractions.ManagedIdentityOptions.#ctor" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>Microsoft.Identity.Abstractions</AssemblyName>
+        <AssemblyVersion>5.1.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters />
+      <Docs>
+        <summary>To be added.</summary>
```

Comment:
Are these fields something we need to fill out?

Reply:
@JoshLozensky in the source code yes, not in the documentation in this repository 😀 Happy to help craft better ref docs in your code if you need any pointers!